packages/boot/tools/supports.ts

```diff
@@ -26,6 +26,7 @@ import {
 import type { ExecutionContext as AvaT } from 'ava';
 
 import { makeRunUtils } from '@agoric/swingset-vat/tools/run-utils.js';
+import { CoreEvalSDKType } from '@agoric/cosmic-proto/dist/codegen/agoric/swingset/swingset';
```

Comment:
Ideally:
```suggestion
import { CoreEvalSDKType } from '@agoric/cosmic-proto/agoric/swingset/swingset';
```

Reply:
That would be nice. Unfortunately it will only work with `NodeNext` [moduleResolution](https://www.typescriptlang.org/tsconfig#moduleResolution) and enabling that breaks our use of ambient types (3364 errors).

On the plus side, it also makes it so we can import from `types-ambient.js` because it treats those files as modules even though they don't have exports. So once we can use https://github.com/microsoft/TypeScript/pull/57207 to import types into the module, the lift will be much lighter.